src/components/Select/Select.tsx

```diff
@@ -29,36 +40,61 @@ export interface SelectProps {
   onBlur?: (event: React.FormEvent) => void;
   /** Passes selects' current value to the React state handler */
   onChange?: (event: React.FormEvent) => void;
+  /** Will add 'aria-required: true' to input */
```

Comment:
You should add that it also adds the native `required` attribute to the input (since browsers handle that differently).

Reply:
Done.